utils/obsproc/util.h

```diff
@@ -238,16 +238,18 @@ namespace gdasapp {
           for (int i = 0; i < location_; i++) {
             if (datetime_(i) < minDAwindow) {
               int delta_t = minDAwindow - datetime_(i);
-              datetime_(i) = minDAwindow + 1;
               // one second is used for safety falling in min DA window
-              obsError_(i) += errRatio * delta_t;
+              datetime_(i) = minDAwindow + 1;
+              // Convert errRatio from day to sec
+              obsError_(i) += (errRatio / 86400) * delta_t;
```

Comment:
No a huge deal, but why not just convert right after reading instead of dividing nobs times?

Reply:
> No a huge deal, but why not just convert right after reading instead of dividing nobs times?

Yes, it should- thanks!